include/RandLAPACK/comps/qb.hh

```diff
@@ -0,0 +1,105 @@
+#ifndef BLAS_HH
+#include <blas.hh>
+#define BLAS_HH
+#endif
+
+#include "orth.hh"
+#include "rf.hh"
+
+namespace RandLAPACK::comps::qb {
+
+#ifndef QB_CLASS
+#define QB_CLASS
+
+template <typename T>
+class QBalg
+{
+        public:
+                virtual void call(
+                        int64_t m,
+                        int64_t n,
+                        std::vector<T>& A,
+                        int64_t& k,
+                        int64_t block_sz,
```

Comment:
Block size isn't part of the call function for ``QBDecomposer`` in Python. Did we decide to keep it here out of an idea that unblocked QB isn't worth bothering with?